NOTICE

```diff
@@ -1,5 +1,5 @@
 {project name} - FINOS
-Copyright {yyyy} {name of copyright owner} {email of copyright holder}
+Copyright {start_year} - {current_year} {name of copyright owner} {email of copyright holder}
```

Comment:
This copyright notice is incorrect.  [Copyright notices only include the year of first publication](https://en.wikipedia.org/wiki/Copyright_notice#Form_of_notice_for_visually_perceptible_copies) - they are inherently open ended from that year forward.